P/POMDPSolve/build_tarballs.jl

```diff
@@ -0,0 +1,56 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "POMDPSolve"
+version = v"5.4.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/JuliaPOMDP/pomdp-solve.git", "9e1e9987e459732c4b435387a346515afcab4e01")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd pomdp-solve/
+
+autoreconf -f -i
+
+./configure --prefix=${prefix} --exec_prefix=${prefix} --build=${MACHTYPE} --host=${target}
+
+make -j${nprocs} CC=${CC} CXX=${CXX} LINKER=${CXX} CFLAGS=${CFLAGS} CXXFLAGS=${CXXFLAGS} LDFLAGS=${LDFLAGS}
+
+make install
+
+mkdir -p ${prefix}/share/licenses/POMDPSolve
+cp COPYING ${prefix}/share/licenses/POMDPSolve/
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libcs="glibc"),
+    Platform("x86_64", "linux"; libc="glibc"),  
+    Platform("x86_64", "macos"),
+    Platform("aarch64", "macos"),
+    Platform("i686", "windows"),
+    Platform("x86_64", "windows"),
+]
+platforms = expand_cxxstring_abis(platforms)
```

Comment:
```suggestion
```
It looks like the software is pure C, so there shouldn't be a need to expand the C++ ABIs in it since `std::string` isn't used anywhere.